game-core/src/main/java/games/strategy/engine/framework/startup/ui/RemoteHostUtils.java

```diff
@@ -110,7 +110,7 @@ public String shutDownHeadlessHostBot(final String hashedPassword, final String
     if (instance == null) {
```

Comment:
Not suggesting this be done in this PR--because it would probably require additional testing--but now that `remoteShutdown()` and `getSalt()` are static, these `instance == null` checks seem to no longer be necessary.  They appear to have only been added to prevent an NPE.  However, if they are truly required, the checks should be hidden in the static methods themselves.